src/utils/FileEmitter.ts

```diff
@@ -24,6 +24,8 @@ interface OutputSpecificFileData {
 	bundle: OutputBundleWithPlaceholders;
 }
 
+const isBrowser = typeof window !== 'undefined';
```

Comment:
I would like to be able to run the browser build of rollup within deno (https://github.com/denoland/deno). Unlike node, deno defines `window`. Could you change this condition to `typeof document !== 'undefined'`?

Reply:
Alternatively, because the type of `source` is `string | Buffer` you could just omit the `Buffer.isBuffer` calls. Also, perhaps the use of the Buffer type should be replaced with Uint8Array to match rollup's public types.

https://github.com/rollup/rollup/blob/master/src/rollup/types.d.ts#L130

Reply:
I added changes along these lines